src/actions/commentActions.js

```diff
@@ -0,0 +1,36 @@
+import { toastr } from 'react-redux-toastr';
+import { globalLoading, globalFailure } from './globalActions';
+import fetchArticle from './singleArticleActions';
+import requestOptions from '../utils/requestOptions';
+import getToken from '../utils/getToken';
+
+export const postComment = (articleId, body) => (dispatch) => {
+  dispatch(globalLoading(true));
+  const commentBody = {
+    body
+  };
+  return fetch(
+    `${process.env.API_BASE_URL}/articles/${articleId}/comments`,
+    requestOptions(commentBody, 'POST', getToken())
+  ).then(
+    res => res.json(),
+    error => dispatch(globalFailure(error))
+  ).then((commentResponse) => {
+    if (commentResponse.status === 'success') {
+      // successfully posted article
+      toastr.success(commentResponse.message);
+      dispatch(fetchArticle(articleId));
+    }
+    if (commentResponse.status === 'unauthorized') {
+      // no token provided or invalid token
+      toastr.error(commentResponse.message);
+    }
```

Comment:
Are we displaying `invalid token` to the user: do they know what a `token` is? Make this error message more descriptive.

Reply:
Good spot. I have made this more descriptive to the user now.